client/x/evmstaking/keeper/keeper_test.go

```diff
@@ -199,6 +180,17 @@ func (s *TestSuite) TestValidatorAddressCodec() {
 	require.NoError(err)
 }
 
+//nolint: dupword // test code
```

Comment:
Instead of commenting out all test cases, I think we can just comment out the case of `pass: process CreateValidatorEvent`, which is the case that we cannot mock the keeper function expectedly.

Reply:
And let's find a way to mock the function which uses cached context internally.

Reply:
yeah agree, and already uncomment the whole Test func